integration/shim_dial_unix_test.go

```diff
@@ -1,5 +1,5 @@
-//go:build !windows
```

Comment:
- This file is having some tests for normal Unix domain sockets, which could run on macOS.
- Moreover, *abstract* Unix domain sockets are only for Linux. So the build flags should be "if Linux" rather than listing non-Linux OSes.